pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -811,16 +810,18 @@ def cmd_class_tree(context, classname, options):
     subclass tree is displayed.
     """
 
-    # TODO: Sort out how we handle output format with tree output.
+    # TODO FUTURE: Sort out how we handle output format with tree output.
     try:
         if options['superclasses']:
             if classname is None:
                 raise click.ClickException('CLASSNAME argument required for '
                                            '--superclasses option')
 
-            # get the superclasses into a list
+            # Get the superclasses into a list
             class_ = context.conn.GetClass(classname,
                                            namespace=options['namespace'])
+            # get correct case sensitive classname
+            classname = class_.classname
```

Comment:
Setting `classname` at this point does not do anything. It is not used until it is set to None a few lines down. Am I missing anything here?

Reply:
You are correct dead code, removed.

Reply:
ok